build/build.yml

```diff
@@ -165,6 +212,32 @@ steps:
     archiveFile: '$(Build.ArtifactStagingDirectory)\$(BuildConfiguration)\Microsoft.NuGet.CredentialProvider.zip'
     replaceExistingArchive: true
 
+- task: CopyFiles@2
+  displayName: Copy netfx 4.8.1 files into tarball
+  inputs:
+    sourceFolder: '$(Build.SourcesDirectory)\CredentialProvider.Microsoft\bin\$(BuildConfiguration)\net481\'
+    contents: '**\*'
+    targetFolder: '$(Build.ArtifactStagingDirectory)\tarball\plugins\netfx\CredentialProvider.Microsoft\'
```

Comment:
I noticed the dotnet x.x copy to tarball steps are cleaning the folder before copying. [The PR run of this pipelines](https://dev.azure.com/mseng/PipelineTools/_build/results?buildId=29152595&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=056a8941-e7c5-5432-3fb1-9fa8318ef3a8) looks like the netfx files are throwing an already exists. 

Is this intentional?